azure-devops/azext_devops/dev/team/arguments.py

```diff
@@ -106,6 +106,28 @@ def load_team_arguments(self, _):
         context.argument('relationship', arg_type=get_enum_type(_RELATIONSHIP_TYPES),
                          help='Get member of/members for this group.')
 
+    with self.argument_context('devops security permission') as context:
+        context.argument('namespace_id', options_list=('--namespace-id', '--id'),
+                         help='ID of security namespace')
+        context.argument('token',
+                         help='Security token.')
+        context.argument('subject',
+                         help='User Email ID or Group descriptor')
+
+    with self.argument_context('devops security permission add') as context:
+        context.argument('merge', arg_type=get_three_state_flag(),
+                         help='If set, the existing ACE has its allow and deny merged with \
+                         the incoming ACE\'s allow and deny. If unset, the existing ACE is displaced.')
+        context.argument('allow_bit', type=int,
```

Comment:
[nit suggest]: Complementary params - Should it be --bit-value as input and --bit-type required with values [allow, deny]. 
PM call.

Reply:
How would we assign allow permissions to few types and deny to few others in that case?

Reply:
@geverghe  , Thoughts?

Reply:
I like the approach suggested by @atbagga as it is clean - you do one thing at a time with the command. However, I would still continue with the current approach - I do not have a strong rationale other than it helps us utilize the full capability offered by the API enabling the scenario highlighted by @ishitam8.

Reply:
Keeping this as is.